StortSpelProjekt/Engine/src/ECS/SceneManager.h

```diff
@@ -14,21 +16,32 @@ class SceneManager
 	SceneManager(Renderer* r);
 	~SceneManager();
 
+	void Update(double dt);
+	void RenderUpdate(double dt);
+
 	Scene* CreateScene(std::string sceneName);
 
+	std::vector<Scene*>* GetActiveScenes();
 	Scene* GetScene(std::string sceneName) const;
 
-	void RemoveEntity(Entity* entity);
-	void AddEntity(Entity* entity);
-	void SetScene(Scene* scene);
+	void RemoveEntity(Entity* entity, Scene* scene);
+	void AddEntity(Entity* entity, Scene* scene);
+	void SetScenes(unsigned int numScenes, Scene** scene);
+	void LoadScene(Scene* scene);
+	void UnloadScene(Scene* scene);
+	void ResetScene();
 private:
 	Renderer* m_pRenderer;
 
 	std::map<std::string, Scene*> m_pScenes;
```

Comment:
Not actually a pointer, no p in variabel name necessary

Reply:
Removed m_pRenderer, now uses Renderer::GetInstance()

Reply:
My comment actually referred to m_pScenes, which should be just m_Scenes